src/Features/TaskTemplates/Header/VersionSwitcher/VersionSwitcher.tsx

```diff
@@ -33,26 +34,31 @@ const VersionSwitcher: React.FC<VersionSwitcherProps> = ({ revisions, currentRev
   };
 
   const renderBackButtons = (enabled: boolean) => {
+    const shouldNotDisplay = !enabled || !canEdit;
```

Comment:
Is the idea here that if you can't edit a TaskTemplate you can't look at the version history?

Reply:
I suppose that you would be fine to still navigate between versions and view history, good point. The idea is just to prevent any changes when the setting to edit is not enabled.